src/test/java/com/karankumar/bookproject/ui/statistics/util/StatisticsViewTestUtils.java

```diff
@@ -91,18 +111,32 @@ public static void populateDataWithBooksWithoutRatings(
     }
 
     private static PredefinedShelf getReadShelf(PredefinedShelfService predefinedShelfService) {
-        PredefinedShelfUtils predefinedShelfUtils = new PredefinedShelfUtils(predefinedShelfService);
+        PredefinedShelfUtils predefinedShelfUtils =
+                new PredefinedShelfUtils(predefinedShelfService);
         return predefinedShelfUtils.findReadShelf();
     }
 
-    private static Book createMobyDickBook(PredefinedShelfService predefinedShelfService) {
+    private static Book createBook(String title, Author author,
+                                   PredefinedShelfService predefinedShelfService, BookGenre genre,
+                                   int numberOfPages, int pagesRead, RatingScale rating) {
```

Comment:
Since this method has lots of parameters, I think we can simplify this. Could you please remove the `genre`, the `numberOfPages`, `pagesRead` and `rating` parameters? (I appreciate 'Moby Dick' may not be fantasy, but I think the code is cleaner this way

Reply:
Done with 3d10c04672fd1d1d0bae625eab3d0562e2f66603